packages/clerk-js/src/ui/components/SignIn/SignIn.tsx

```diff
@@ -132,12 +136,27 @@ function SignInRoutes(): JSX.Element {
                 <LazySignUpContinue />
               </Route>
             </Route>
+            {SESSION_TASK_PATHS.map(path => (
+              <Route
+                path={path}
+                key={path}
+              >
+                <LazySignUpSessionTask />
+              </Route>
+            ))}
             <Route index>
               <LazySignUpStart />
             </Route>
           </Route>
         )}
-
+        {SESSION_TASK_PATHS.map(path => (
```

Comment:
There's an curious behavior happening here, in which might be related with the routing structure, but I wanted to comment here anyway 

When we navigate to next factors, the DOM gets repainted on the exact nodes, but the whole page gets re-rendered for the task navigation (you can see the diffs below) 

Our Dashboard contains the following snippet: 

```
<div className='relative z-20 [--animation-duration:250ms] [&>div]:animate-fade-in'>
                <SignIn
                  routing='path'
                  path='/sign-in'
```

The above animation gets triggered again and it causes this small flickering on navigation: 


https://github.com/user-attachments/assets/04d6fedc-3899-40bd-93bf-167e7ae5d25e